Reverting https://gitlab.gnome.org/GNOME/glib/-/commit/5f38ae5f
since it results in full UL build failures in several components
that use studio to build.

"/usr/include/glib-2.0/glib/gmacros.h", line 562: unexpected "("

Reverting to what we have with the existing version of glib

Not suitable for upstream

--- a/glib/gmacros.h	2020-09-26 06:30:09.767195634 -0700
+++ b/glib/gmacros.h	2020-09-26 06:31:03.323332828 -0700
@@ -559,8 +559,6 @@
  */
 #if    __GNUC__ > 6
 #define G_GNUC_FALLTHROUGH __attribute__((fallthrough))
-#elif g_macro__has_attribute (fallthrough)
-#define G_GNUC_FALLTHROUGH __attribute__((fallthrough))
 #else
 #define G_GNUC_FALLTHROUGH
 #endif /* __GNUC__ */
